zips/zip-0235.md

```diff
@@ -0,0 +1,151 @@
+We propose to burn 60% of transaction fees, while the remaining 40% be directed
+as before, providing a deflationary effect, and building the groundwork for
+long-term support of the Zcash network via the new block subsidy rules proposed
+by ZIP-234.
+
+# Motivation
+
+ZIP-233 ("Network Sustainability Mechanism: Burning") describes a method by
+which ZEC can be burned to support network sustainability.
+
+By introducing a requirement that a certain proportion of transaction fees be
+burned, we ensure that ZEC will be removed from circulating supply to contribute
+to the long-term sustainability of the network as described below:
+
+## Benefits to the Network
+
+1. **Network Sustainability**: This mechanism involves temporarily reducing the
+   supply of ZEC, similar to asset burning in Ethereum’s EIP-1559, but with
+   long-term sustainability benefits, as the burned funds effectively boost
+   future mining rewards, making it an attractive option for current and future
+   Zcash users.
+2. **Incentivizing Transaction Inclusion**: By maintaining a 40% share of
+   transaction fees for miners, we continue incentivizing miners to prioritize
+   including transactions in their blocks. This helps ensure the continued
+   efficient processing of transactions and supports a robust and responsive
+   network.
+3. **Aligning Interests:** Burning transaction fees aligns the interests
+   of miners with the long-term health of the network, incentivizing them
+   to prioritize security and efficiency. As miners focus on maintaining a
+   robust network, overall adoption and usage can increase, leading to higher
+   transaction volumes in the long run. This structure discourages short-term
+   profit-seeking behaviors, as miners benefit more from a stable and thriving
+   ecosystem than from engaging in malicious activities that could undermine the
+   network's reputation and security.
+4. **Future-Proofing the Network**: Burning a portion of transaction fees
+   is a forward-looking approach that prepares the Zcash network for future
+   challenges and opportunities. It establishes a financial buffer that can be
+   instrumental in addressing unforeseen issues and seizing strategic advantages
+   as the Zcash ecosystem evolves.
+
+# Requirements
+
+* For each block, at least 60% (rounded down) of the total fees are to be
+burned.
+* No restrictions are placed on the destination of the remaining proportion of
+fees.
+* Any fractions are rounded in favor of the miner.
+
+# Specification
+
+## Consensus Rule Changes
+
+For a given block, the coinbase transaction MUST have a `burnAmount` that is
+greater than or equal to `floor(transactionFees * 6 / 10)`.
+
+Previous transaction versions are not supported for coinbase transactions, due
+to there being no explicit mechanism to burn the required funds.
+
+# Deployment
+
+The implementation of this ZIP MUST be deployed at the same time or after
+ZIP-233 ("NSM: Burning"), and ZIP-234 ("NSM: Issuance Smoothing").
```

Comment:
Is it intended that this be dependent on ZIP-234, or is that incidental?

Reply:
Also, if that is intended, does it mean at the same time or after NU7 activation, or at the same time or after the smoothing takes effect (which may be later).

Reply:
This can be resolved during the editorial period.

Reply:
Yes, if possible please let's discuss this during the editorial period. However, ZIP 235 isn't intended to be dependent on ZIP 234; it can exist independent of ZIP 234, but benefits from the voluntary burning mechanism in ZIP 233. Also, it's worth noting here that at the last Arborist call, we talked about this not being a consensus-level change and instead being a node default behavior (or opt-in).  We should discuss that further during the editorial period.